draft-ietf-quic-transport.md

```diff
@@ -961,12 +961,15 @@ cannot expect its peer to store and use all issued connection IDs.
 An endpoint SHOULD ensure that its peer has a sufficient number of available and
 unused connection IDs.  While each endpoint independently chooses how many
 connection IDs to issue, endpoints SHOULD provide and maintain at least eight
-connection IDs.  The endpoint SHOULD do this by always supplying a new
-connection ID when a connection ID is retired by its peer or when the endpoint
-receives a packet with a previously unused connection ID.  Endpoints that
-initiate migration and require non-zero-length connection IDs SHOULD provide
-their peers with new connection IDs before migration, or risk the peer closing
-the connection.
+connection IDs.  The endpoint SHOULD do this by supplying a new connection ID
+when a connection ID is retired by its peer or when the endpoint receives a
+packet with a previously unused connection ID, though it MAY limit the frequency
+or the total number of connection IDs issued for each connection to avoid the
+risk of running out of connection IDs (see {{reset-token}}).
+
+Endpoints that initiate migration and require non-zero-length connection IDs
+SHOULD provide their peers with new connection IDs before migration, or risk the
+peer closing the connection.
```

Comment:
Now that I read this paragraph again, I think that we need to rework it, but that's a separate problem.